044-kraft-liveness-readiness.md

```diff
@@ -0,0 +1,174 @@
+# Liveness and Readiness probes in a KRaft Kafka cluster
+
+This proposal describes the liveness and readiness probes that should be put in place 
+for a Kafka cluster that is using KRaft rather than ZooKeeper. This includes a KRaft 
+cluster with combined nodes and one that has separate controller and broker nodes.
+
+## Current situation
+
+These are the current probes used in Strimzi when ZooKeeper is present:
+
+|Component| Liveness | Readiness |
+|---|---|---|
+| ZooKeeper | "imok" from 127.0.0.1:12181 | "imok" from 127.0.0.1:12181 |
+| ZK-based broker | if (ready) { listening on 9091 } else { have ZK session } | if (ready) { return true } |
+
+The "ready" check in both cases is based on the existence of the file `../kafka-ready` which is 
```

Comment:
Maybe you can clarify that both cases here means `Kafka liveness and readiness probe` and `ZooKeeper and Kafka` => I think it can be a little bit confusing with having both Kafka and Zoo in the Table right above.